cli/cmd/vuln_container.go

```diff
@@ -65,6 +70,50 @@ Arguments:
 		},
 	}
 
+	// vulContainerListRegistriesCmd represents the list-registries sub-command inside the container
+	// vulnerability command
+	vulContainerListRegistriesCmd = &cobra.Command{
+		Use:     "list-registries",
+		Aliases: []string{"list-reg", "registries"},
+		Short:   "list all Container Registries configured",
+		Long:    `List all Container Registries configured in your account.`,
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, args []string) error {
+			registries, err := getContainerRegistries()
+			if err != nil {
+				return err
+			}
+			if len(registries) == 0 {
+				msg := `There are no Container Registries configured in your account.
```

Comment:
@afiune  is there a reason why we capitalize `Container Registries`?

Reply:
no, I will change it